src/geo/projection/globe.js

```diff
@@ -315,7 +313,7 @@ export function globePoleMatrixForTile(id: CanonicalTileID, south: boolean, tr:
         mat4.scale(poleMatrix, poleMatrix, [1, -1, 1]);
     }
 
-    return poleMatrix;
+    return Float32Array.from(poleMatrix);
```

Comment:
Same as above — this is intentional because we expect to use it as a `Float32Array` type, but start with `Float64Array` so that intermediary calculations are precise.